addons/project/models/project.py

```diff
@@ -943,6 +963,11 @@ def _notify_email_header_dict(self):
         return headers
 
     def _message_post_after_hook(self, message, msg_vals):
+        if message.attachment_ids and not self.displayed_image_id:
+            image_attachments = message.attachment_ids.filtered(lambda a: a.mimetype == 'image')
```

Comment:
Hello, I just noticed this code never worked and it never will as `image` is not a valid mimetype.
Should it be fixed, or removed?